Brewfile

```diff
@@ -1,7 +1,7 @@
 brew "gh"
 brew "ghostscript"
 brew "imagemagick"
-brew "pdftk-java"
+brew "java"
```

Comment:
our `gyr_efiler_spec` has previously been suggesting you install Java like this:
`brew tap AdoptOpenJDK/openjdk && brew install adoptopenjdk8`

is `brew "java"` better? worse? the same?

Reply:
never mind i see the old one is just for the efiler